.travis.yml

```diff
@@ -0,0 +1,34 @@
+language: python
+
+python:
+  - 3.6
+addons:
+  postgresql: "9.4"
+
+services: postgresql
+
+env:
+  - DJANGO=2.1.7 DB=postgresql
+
+install:
+  - pip install -r api/requirements.txt
+  - pip install coveralls
+
+
+before_script:
+  - psql -c "create user django WITH PASSWORD 'password';" -U postgres
+  - psql -c "create database drf;" -U postgres
+
+script:
+- python api/manage.py makemigrations
+- python api/manage.py migrate
+- python api/manage.py test users
+- coverage run api/manage.py test
```

Comment:
Ignore files that are in the `virtualenv` from being calculated for coverage.
![image](https://user-images.githubusercontent.com/41262970/53870390-32a4f480-400b-11e9-82a0-3210dfbe1ae5.png)

It is showing over 700 entries. Ignore the files what we did not write code for.

Reply:
Yes worked on it

Reply:
@davenjeru Check the branch, worked on the feedback